main.js

```diff
@@ -209,12 +211,19 @@ var $search = document.querySelector('#search-top')
 var $searchCompare = ''
 var $leaderboard = document.querySelector('#leader-board')
 var $playerStats = document.querySelector('#player-stats')
+var $siteLogo = document.querySelector('#site-logo')
+
+$siteLogo.addEventListener('click', writeLeaderBoard)
 
 function removeElement(element) {
   element.className += ' hidden'
   element.innerHTML = ''
 }
 
+function showElement(element) {
+  element.className -= ' hidden'
```

Comment:
Does this actually work?

Reply:
It might give you the result you want, but I don't think it's doing what you expected.